_includes/docs/0.4.x/cloud-foundry.html

```diff
@@ -40,6 +40,11 @@
         <li><pre>$ grunt build</pre></li>
       </ul>
     </li>
+    <li>Run the Gulp Build task to build the optimized JavaScript and CSS files (If you chose Gulp instead of Grunt)
```

Comment:
@codydaig I'm removing this line and sending a new commit now. Will wait for 0.5 to add.